roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/RoqFrontMatterConfig.java

```diff
@@ -49,9 +49,15 @@ public interface RoqFrontMatterConfig {
     /**
      * Format for dates
      */
-    @WithDefault("yyyy-MM-dd HH:mm:ss Z")
+    @WithDefault("yyyy-MM-dd[ HH:mm][:ss][ Z]")
     String dateFormat();
 
+    /**
+     * The default timezone
+     */
+    @WithDefault("UTC")
+    String timeZone();
```

Comment:
I would make it optional and use System as default, any reason for choosing UTC by default?